test/python/tests/test_defaults.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from rosparam_handler.param.DefaultsParameters import DefaultsParameters
+
+class TestDefaults(unittest.TestCase):
```

Comment:
Add the test for defaults on param server after fromServer(). (PR #22 is merged)